frontend/packages/dev-console/src/components/topology/topology-utils.ts

```diff
@@ -218,7 +218,7 @@ export class TransformTopologyData {
         data: {
           url: !_.isEmpty(route.spec) ? getRouteWebURL(route) : this.getRouteData(ksroute),
           kind: targetDeploymentsKind,
-          editUrl: deploymentsAnnotations['app.openshift.io/edit-url'],
+          editUrl: deploymentsAnnotations['app.openshift.io/vcs-uri'],
```

Comment:
Let the user specify `edit-url` to refer to a code editor and fallback to the source url.
```suggestion
          editUrl: deploymentsAnnotations['app.openshift.io/edit-url'] || deploymentsAnnotations['app.openshift.io/vcs-uri'],
```

Reply:
Okay